### ISSUE TITLE
startRecordingScreenAndAudio in example app only records video for 1 second and audio is not captured.

### ISSUE THREAD

**Issue by Fairfieldfred** [Isvisoft/flutter_screen_recording#112]
In the repository example app FlutterScreenRecording.startRecordingScreen() works just fine but FlutterScreenRecording.startRecordScreenAndAudio() will only capture one second of video.
At least in iOS.

**Comment by goldsergeant** [comment]
me too.

**Comment by jackobst** [comment]
yes , screen record only 1 section and application in background state screen recording not capture and returned file courpted
